src/naksu/mebroutines/runandgetoutput_windows.go

```diff
@@ -0,0 +1,58 @@
+package mebroutines
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+	"syscall"
+
+	"naksu/log"
+)
+
+func escapeWindowsCommandArgs(args []string) []string {
```

Comment:
Täsmällisempi nimi tälle vois olla `quoteWindowsCommandArgs`. En oo kuullut, että escape-verbiä käytettäis muuhun kuin `\`-tyyppiseen escapointiin.

Reply:
Joo!